src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-router.js

```diff
@@ -50,15 +43,60 @@ define([
         toRootCollection: function(locale) {
             locale = locale || UserSettingsManager.getMediaLocale();
             Mediator.emit('sulu.router.navigate', 'media/collections/' + locale, true, true);
```

Comment:
Isn't the advantage of this pattern that you don't have to hardcode the url here?